src/Sarif/Sarif.csproj

```diff
@@ -39,6 +39,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="GlobalSuppressions.cs" />
```

Comment:
This PR didn't introduce a GlobalSuppression file (I think Kevin did that yesterday, right?). Do you have a merge problem? In any case, you won't be introducing a new .resx file, so you can just revert the changes to this project file.